### ISSUE TITLE
More intuitive write operation syntax

### ISSUE THREAD

**Issue by carlobeltrame** [ecamp/hal-json-vuex#1]
### Current
```Javascript
this.api.patch(book, { title: "New book title" }).then(...)
createdBook = await this.api.post(books, { title: "new book" })
```

### Proposed
```Javascript
book.$patch({ title: "New book title" }).then(...)
createdBook = await books.$post({ title: "new book" })
```

_Originally posted by @usu in https://github.com/ecamp/ecamp3/issues/507#issuecomment-723483312_

**Comment by pmattmann** [comment]
Auf Entity:
- $reload : Promise
- $patch : Promise
- $delete : Promise

Auf Colleciton:
- $post : Promise
- $patch : Promise  (evtl für Listen sortieren)


Idee auf Collection:
- $load für LinkColleciton, lädt alle Links und Resolved dann (Print)

**Comment by carlobeltrame** [comment]
Fixed in #10, will soon be released
